src/hook/mydream/useMdIsPublicMutation.ts

```
@@ -0,0 +1,38 @@
+      todoGroupId: number;
+      isPublic: boolean;
+    }) => {
+      try {
+        const { data } = await api.patch(
+          `/v1/my-dream/todo/${todoGroupId}/public-state`,
+          { isPublic }
+        );
+
+        return data.data as TodoPublicResponse;
+      } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)